cmd.rb

```diff
@@ -14,6 +14,6 @@
          'The wanted resolutions in order (Default: 1080p,720p') { |o| options.resolutions = o }
 end.parse!
 
-m = MovieCleaner.new(options.base_path, options.verbose, options.resolutions)
-p m.sub_par_files
-m.print_rm_commands
+movie_cleaner = MovieCleaner.new(options.base_path, options.verbose, options.resolutions)
```

Comment:
Interesting take on the whole variable naming debate. Could you elaborate the reasoning behind this?

Reply:
In order for new contributors to easily understand and take an active role in the further development of this project ([gem?](https://github.com/skius/movie-cleaner/issues/13)) the variable names should be clear and easy to read. "m" is a very bad example of that as it is ambigous of what you are actually trying to say.
Thank you for supporting the vibrant open source community!

Reply:
I totally agree with @TheFehr. As an independent software quality agent, I assure that this is a major improvement over the previous version.